### DISCUSSION TITLE
what might be the reason for segfault ?

### DISCUSSION THREAD

**Discussion by imxade** [termux/proot#290] [Q&A] [answered]
## what I did ? & How to reproduce ?
- https://codeberg.org/zz/strap/wiki#on-termux

## script
- https://codeberg.org/zz/strap/src/branch/master/pacstrap.sh

**Comment by michalbednarski** [comment, marked as answer]
There was error in proot that prevented launching native (ARM) executables from QEMU (x86_64), fixed now and will be available as `proot-5.1.107-62` in Termux apt repo

Your script sets `PATH=/data/data/com.termux/files/usr/bin:/usr/local/sbin:/usr/local/bin:/usr/bin` and therefore binaries from Termux are preferred to ones in guest system, not sure if that is intentional

By the way, you can avoid `linkerconfig` warning through:

```
mkdir directory/linkerconfig
cp /linkerconfig/ld.config.txt directory/linkerconfig/
```
